cdc/sink/mysql/mysql.go

```diff
@@ -214,7 +218,12 @@ func NewMySQLSink(
 	if err != nil {
 		return nil, err
 	}
-	go sink.flushRowChangedEvents(ctx, receiver)
+
+	sink.workerWg.Add(1)
+	go func() {
+		sink.flushRowChangedEvents(ctx, receiver)
+		sink.workerWg.Done()
```

Comment:
It is better to call wg.Done() with a defer, since the defer function will always be called, even `panic` or `runtime.Goexit` happens in `sink.flushRowChangedEvents`
```go
defer sink.workerWg.Done()
sink.flushRowChangedEvents(ctx, receiver)
```

Reply:
Thank for your advice~